application-admintools-ui/src/main/resources/AdminTools/Code/AdminToolsJS.xml

```diff
@@ -271,6 +271,23 @@
       }
     }
   }
+
+  $(document).on('click', '#confirmCacheFlushModal .btn-primary', function(event) {
+    event.preventDefault();
+    const flushButton = $('#confirmCacheFlushModal .btn-primary');
+    const target = flushButton.attr('data-action')
+    $.ajax({
+      url: target,
+      type: 'POST',
+      success: function(response) {
+        $(".operationMessage").html('&lt;div class="successmessage"&gt;'+ response + '&lt;/div&gt;');
```

Comment:
It would be better construct the success message from here, instead of sending it from the rest endpoint
[Here](https://github.com/xwikisas/application-collabora/blob/main/application-collabora-ui/src/main/resources/Collabora/Code/Main.xml#L237) you have an example for how the `collabora.editor.error` translation was used in JS